.github/config/labeler-config.yml

```diff
@@ -0,0 +1,7 @@
+# Pull Request Labeler Github Action Configuration: https://github.com/marketplace/actions/labeler
```

Comment:
Note that labels get added "after" the PR gets created and hence the CI run won't see the labels most probably - can we test this? (subsequent pushes would see the labels though).

Reply:
Yes, this is what i expect. This automation doesn't free us from thinking.

However, oftentimes PRs do involve multiple pushes, so this may free us from doing the thing manually.

Reply:
There's still one issue I see - a reviewer won't know whether the hive tests actually ran just by looking at presence of label.
I'm fine with merging this provided people know of the behaviour to expect from the label.

Reply:
Not sure if it is already discussed, what if we use [workflow_run](https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#workflow_run) so that we could run this workflow could add labels and ci could use them ?

Reply:
> There's still one issue I see - a reviewer won't know whether the hive tests actually ran just by looking at presence of label.

If you see a label added after the PR is created, you must not assume it was observed by the workflow run.

But, if there was a push since label was added, then you can assume it's observed